### ISSUE TITLE
About training loss

### ISSUE THREAD

**Issue by YangGangZhiQi** [Yang-Liu1082/InvDN#25]
Hi, thanks for this great works! I try to train invdn in my own datasets. I get a strange problem. The l_back_rec is too big, eg 1e+3. I don't know whether it is normal. 
![InvDN_back_rec_loss太大](https://user-images.githubusercontent.com/87355944/209607203-5fa226d2-b63f-4398-a1e8-2b7f8fc8478b.png)
I would apprecaite it if someone could help me.

**Comment by YangGangZhiQi** [comment]
Today I figure it out. Because the loss function calculates the sum of  x-target, so the results is e*C*H*W。For example, C=3, H=W=144, average error e=0.5, the loss result=0.5*3*144*144=3.1104e+4. So the loss results above is normal.

**Comment by 8056** [comment]
Thank you, could you mind to share the final loss during your trainning ?

**Comment by sui1999** [comment]
> 今天我想通了。因为损失函数计算的是x-target的总和，所以结果是e _C_ H _W。比如C=3，H=W=144，平均误差e=0.5，损失结果=0.5_ 3 _144_ 144=3.1104 e+4。所以上面的损失结果是正常的。

Hi ,May I ask how you configured the code after downloading it? I don't quite understand how my training is particularly slow
